demo/api/schema.gql

```diff
@@ -543,6 +558,7 @@ type Query {
   damItemListPosition(sortColumnName: String, sortDirection: SortDirection! = ASC, scope: DamScopeInput!, id: ID!, type: DamItemType!, folderId: ID, includeArchived: Boolean, filter: DamItemFilterInput): Int!
   damFilesList(offset: Int! = 0, limit: Int! = 25, sortColumnName: String, sortDirection: SortDirection! = ASC, scope: DamScopeInput!, folderId: ID, includeArchived: Boolean = false, filter: FileFilterInput): PaginatedDamFiles!
   damFile(id: ID!): DamFile!
+  getAllFilesUsedOnPage(pageTreeNodeId: ID!): [DamFile!]!
```

Comment:
I don't see a reason why this is a root query instead of a FieldResolver in PageTreeNode

Reply:
I moved it to a FieldResolver